### ISSUE TITLE
Geolocation HTTPS issue

### ISSUE THREAD

**Issue by ArtificialTruth** [mapcentia/gc2collector#19]
~~When trying to get the location a dialog box appears with the message "no location".
Tested on chrome 51.0.2704.81 for Android 5.0.2 with some flags changed. The website is not requesting chrome for the location anymore, even though location services is allowed in chrome settings.
Additionally, it works on the default internet browser.~~

It turns out that starting with Chrome 50, Chrome no longer supports obtaining the user’s location using the HTML5 Geolocation API through unsesure connections. To fix this issue, MapCentia needs to use HTTPS for the entire page. See https://developers.google.com/web/updates/2016/04/geolocation-on-secure-contexts-only
Location works on https://maps.google.com but not on http://ballerup.mapcentia.com/apps/gc2collector/ for instance.
It would be good to show an error if the geolocation was blocked because of not being on a secure context, and this is possible (again, see the [link](https://developers.google.com/web/updates/2016/04/geolocation-on-secure-contexts-only))

**Comment by mapcentia** [comment]
I agree - an alert should pop up when the API is blocked. As a side note, letsencrypt.org is been baked into the GC2 Docker image, so https can be enabled without purchasing a certificate.
